addons/account/migrations/11.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,197 @@
+---Fields in module 'account'---
+account      / account.account          / group_id (many2one)           : NEW relation: account.group
+account      / account.account.tag      / active (boolean)              : NEW 
```

Comment:
@StefanRijnhart Doesn't the analysis now tell about fields with default function?

Reply:
Only for required fields now. I've now created https://github.com/OCA/OpenUpgrade/issues/1204 to track this issue.

Reply:
Thanks!